src/progressive-profiling/tests/ProgressiveProfiling.test.jsx

```diff
@@ -69,7 +69,7 @@ describe('ProgressiveProfilingTests', () => {
     },
   };
   const extendedProfile = ['company'];
-  const optionalFields = { fields, extended_profile: extendedProfile };
```

Comment:
You would need to update it in other tests as well.

Reply:
Done.